pkgs/tools/security/pass/extensions/tail.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "pass-tail";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "palortoff";
+    repo = "pass-extension-tail";
+    rev = "v${version}";
+    sha256 = "RyPXW4Lgx6GdF9j+zEJqKRGxinDwPCmF9WXJ5yW4Pcc=";
+  };
+
+  buildInputs = [ ];
+
+  dontBuild = true;
+
+  installFlags = [
+    "PREFIX=$(out)"
```

Comment:
Isn't this automatically set?

Reply:
True. Removed "dontBuild".

Reply:
`dontBuild = true;` is fine, it's more the `PREFIX=$(out)`, have you tried building this without it?

Reply:
It shows the error that it can't find /usr (of course, the Makefile searchs for it if PREFIX isn't defined (https://github.com/palortoff/pass-extension-tail/blob/master/Makefile#L1))

```
       > Running phase: installPhase
       > install flags: SHELL=/nix/store/agkxax48k35wdmkhmmija2i2sxg8i7ny-bash-5.2p26/bin/bash BASHCOMPDIR=\$\(out\)/share/bash-completion/completions install
       > install: cannot create directory '/usr': Permission denied
```